flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -25,6 +25,9 @@
 import org.apache.flink.configuration.description.Description;
 import org.apache.flink.util.ArrayUtils;
 
+import org.apache.flink.shaded.guava18.com.google.common.base.Splitter;
+import org.apache.flink.shaded.guava18.com.google.common.collect.Iterables;
```

Comment:
I think the decision is that we shouldn't be adding more guava dependencies/usages in our code base. At least that's the feedback once I tried to pull out the same trick you did ("guava already present, so why not to use it?").

Reply:
Turns out people are fine with using more of guava now. 🤷‍♂

Reply:
Discussion came up when the `GuavaFlinkConnectorRateLimiter` was added.

Reply:
Ok, I don't have any strong preference in that matter :)